includes/html/graphs/device/sla_jitter-loss-percent.inc.php

```diff
@@ -10,13 +10,14 @@
  * option) any later version.  Please see LICENSE.txt at the top level of
  * the source code distribution for details.
  */
+use App\Models\Sla;
 
-$sla = dbFetchRow('SELECT `sla_nr` FROM `slas` WHERE `sla_id` = ?', [$vars['id']]);
+$sla_nr = Sla::where('sla_id', $vars['id'])->value('sla_nr');
 
```

Comment:
Just missing the `if ($sla_nr) { .... }` now to make sure we are acting on results from mysql

Reply:
thanks for the pointers @laf. I couldn't see the reason for it at first but think I understand now. I've added it in.

Reply:
styleci doesn't like the closing brackets. I don't know why...